### PR TITLE
Update to v1.60.0

### PR DESCRIPTION
**Pull request by xylar**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/radical_utils-{{ version }}.tar.gz
+  sha256: 9d32aded98ba5464ac6fd7efb4769c024342a884aca824b5b70d9ebd8b81a281
 
 build:
+  noarch: python
```
Comment (xylar) [inline recipe/meta.yaml:13]: I don't see why this couldn't be `noarch: python` but let me know if anyone knows why not.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -51,3 +51,4 @@ extra:
   recipe-maintainers:
     - andre-merzky
     - mtitov
+    - xylar
```
Comment (xylar) [inline recipe/meta.yaml:54]: I hope you don't mind another maintainer.

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
Not sure why the bot didn't make this one.

Needed for https://github.com/conda-forge/radical.pilot-feedstock/pull/121

**Comment by xylar** [comment]
@andre-merzky and @mtitov, I believe this is ready to merge, assuming you're okay with my 2 comments above.

**Comment by andre-merzky** [comment]
Thanks @xylar

**Pull request merged**